### ISSUE TITLE
Install-WslIpHandler:  error in creating scheduled task

### ISSUE THREAD

**Issue by klinge** [wikiped/WSL-IpHandler#13]
Thanks for fixing the last issue I noticed! 

When trying to run the latest version 0.11.4 2a63d12 I now get an error when the script tries to create the scheduled task.. 

```
Install-WslIpHandler -WslInstanceName Ubuntu-20.04 -GatewayIpAddress 198.51.100.1 -WslInstanceIpAddress 198.51.100.1 -UseScheduledTaskOnUserLogOn -DontModifyPsProfile -Verbose
PowerShell installing WSL-IpHandler to Ubuntu-20.04...
VERBOSE: Hyper-V VM Adapter 'WSL' already exists.
VERBOSE: vEthernet (WSL) Network Connection Setup succeeded after [0] seconds.
VERBOSE: Setting Static IP Address: 198.51.100.1 for Ubuntu-20.04.
VERBOSE: Registering WSL-IpHandler scheduled task...
Set-WslScheduledTask: The variable '$commonParams' cannot be retrieved because it has not been set.
```

**Comment by NormannEiland** [comment]
I got the same error when running:
Install-WslIpHandler -WslInstanceName Ubuntu -GatewayIpAddress 172.16.0.1 -WslInstanceIpAddress 172.16.0.2 -UseScheduledTaskOnUserLogOn

**Comment by wikiped** [comment]
Thanks for reporting this. It has been fixed in Version `0.11.5`.
